src/main/scala/ch/openolitor/kundenportal/repositories/KundenportalRepositoryQueries.scala

```diff
@@ -218,14 +219,15 @@ trait KundenportalRepositoryQueries extends LazyLogging with StammdatenDBMapping
       .toManies(
         rs => abotypMapping.opt(aboTyp)(rs),
         rs => lieferpositionMapping.opt(lieferposition)(rs),
-        rs => lieferplanungMapping.opt(lieferplanung)(rs)
+        rs => lieferplanungMapping.opt(lieferplanung)(rs),
+        rs => abwesenheitMapping.opt(abwesenheit)(rs)
       )
-      .map((lieferung, abotyp, lieferposition, lieferplanung) => {
+      .map((lieferung, abotyp, lieferposition, lieferplanung, abwesenheit) => {
         val bemerkung = lieferplanung match {
           case Nil => None
           case x   => x.head.bemerkungen
         }
-        copyTo[Lieferung, LieferungDetail](lieferung, "abotyp" -> abotyp.headOption, "lieferpositionen" -> lieferposition, "lieferplanungBemerkungen" -> bemerkung)
+        copyTo[Lieferung, LieferungDetail](lieferung, "abotyp" -> abotyp.headOption, "lieferpositionen" -> lieferposition, "lieferplanungBemerkungen" -> bemerkung, "anzahlAbwesenheiten" -> abwesenheit.length)
```

Comment:
I'm ok with the change. As said, it undoes the view centric behavior and therefore we must make sure that `anzahlAbwesenheiten` that still exists on the DB is ever used. Therefore:

- [ ] Delete field `anzahlAbwesenheiten` in Lieferung (database and model)
- [ ] Create LieferungDetail featuring  `anzahlAbwesenheiten` as additional field and use it in the two occurrences in this PR

(This can be done in a following taks.)

Reply:
Here https://github.com/OpenOlitor/operations/issues/497